frontend/package.json

```diff
@@ -28,15 +27,16 @@
     "babel-core": "^6.24.0",
     "babel-env": "^2.4.1",
     "babel-loader": "^6.4.1",
-    "babel-preset-es2015": "^6.24.1",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-decorators": "^6.24.1",
     "babel-plugin-transform-react-constant-elements": "^6.23.0",
     "babel-plugin-transform-react-inline-elements": "^6.22.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
babel-preset-es2015 is needed to successfully transpile react-multistep, which has to be done manually by script in the build process